code/control/DNRoot.php

```diff
@@ -88,12 +89,14 @@ class DNRoot extends Controller implements PermissionProvider, TemplateGlobalPro
 		'gitRevisions',
 		'deploySummary',
 		'startDeploy',
+		'createproject',
+		'CreateProjectForm',
```

Comment:
Ah ok thanks, I thought we are still scaffolding. Thanks for explaining the driver.